dbt/parser/schemas.py

```diff
@@ -403,11 +407,16 @@ def parse_model(cls, model, package_name, root_dir, path, root_project,
                                   all_projects, macros)
             yield 'test', node
 
+        context = {'doc': dbt.context.parser.docs(model, docrefs)}
+        description = model.get('description')
```

Comment:
It looks like if the model doesn't have a `description`, we pass `None` into a `ParsedNodePatch` below, resulting in a validation error. Can we use am empty string default here?

```
  File "/Users/drew/fishtown/dbt/dbt/parser/schemas.py", line 448, in load_and_parse
    for result_type, node in v2_results:
  File "/Users/drew/fishtown/dbt/dbt/parser/schemas.py", line 364, in parse_v2_yml
    for node_type, node in iterator:
  File "/Users/drew/fishtown/dbt/dbt/parser/schemas.py", line 416, in parse_model
    patch = ParsedNodePatch(
  File "/Users/drew/fishtown/dbt/dbt/api/object.py", line 37, in __init__
    self.validate()
  File "/Users/drew/fishtown/dbt/dbt/api/object.py", line 85, in validate
    raise ValidationException(msg)
dbt.exceptions.ValidationException: Runtime Error
  Invalid arguments passed to "ParsedNodePatch" instance: description.None is not of type 'string'
```

Reply:
Oh, yes - I thought I got all of these, I'll see if I left any more.